src/components/plots/styling/volcano/MarkersEditor.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { Form } from 'antd';
 import ColorBrowser from '../ColorBrowser';
 
-const MarkersEditor = (props) => {
+const VolcanoMarkersEditor = (props) => {
```

Comment:
rename the file to match the component name

Reply:
renamed to `MarkersEditor`